<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Train Model Test Bench</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="tracButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Model Inputs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="controllerButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Train Controller Inputs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blockButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Update Block</string>
    </property>
   </widget>
   <widget class="QPushButton" name="passButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Update Passengers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tempButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Controller Update Temp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="physButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Physics Calculation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Train</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>390</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Train</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>commanded speed, vital authority</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="trackInput">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>commanded speed, power, service brake, emergency brake,
 left/right doors, announce station, cabin lights, headlights</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="controllerInput">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>170</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>grade, elevation, underground</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="blockInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>incoming</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tempInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>510</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="uiButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>train ui</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trainControllerUpdateButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Train controller update</string>
    </property>
   </widget>
   <widget class="QComboBox" name="trainSelect">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Train select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>beacon</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="beaconInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
